src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -176,27 +190,52 @@ const ProfileCardInner = ({
             ]}
           />
         </div>
-        <span className={classes.name}>
-          {user.name}
+        <div className={classes.name}>
+          <span>{user.name}</span>
           {user.role === USER_ROLE_COORDINAPE && (
-            <ApeInfoTooltip classes={{ tooltip: classes.tooltip }}>
-              <b>Why is Coordinape in my circle?</b>
-              <div>
-                To date Coordinape has offered our service for free. We decided
-                that using the gift circle mechanism as our revenue model might
-                make a lot of sense, so we’re trying that out.
-              </div>
-              <a
-                href={EXTERNAL_URL_FEEDBACK}
-                rel="noreferrer"
-                target="_blank"
-                className={classes.tooltipLink}
-              >
-                Let us know what you think
-              </a>
-            </ApeInfoTooltip>
+            <span
+              style={{
+                marginTop: '8px',
+              }}
+            >
+              <ApeInfoTooltip classes={{ tooltip: classes.tooltip }}>
+                {user.role !== 2 ? <></> : <></>}
+                <b className={classes.tooltipTitle}>
+                  Why is Coordinape in my circle?
+                </b>
+                <p className={classes.tooltipBodyText}>
+                  To date Coordinape has offered our service for free. We
+                  decided that using the gift circle mechanism as our revenue
+                  model might make a lot of sense, so we’re trying that out.
+                </p>
+                <a
+                  href={EXTERNAL_URL_FEEDBACK}
+                  rel="noreferrer"
+                  target="_blank"
+                  className={classes.tooltipLink}
+                >
+                  Let us know what you think
+                </a>
+                <div className={classes.tooltipButton}>
```

Comment:
three things about this:

1. i think this div is unnecessary? there's already one two lines below.
2. the test on the next line is incorrect. you're supposed to be checking that the viewing user is an admin; this checks that the user whose card you're looking at is the Coordinape user.
3. when you want to hide or show a component based on a boolean value, the idiomatic way to do it is `test && <Component>`, not `test ? <Component> : ''`.